src/actions.js

```diff
@@ -257,14 +254,24 @@ export function fetchRegionDistricts(parent) {
   return graphql(payload, `LOCATION_REGION_DISTRICTS`);
 }
 
+export function fetchDataFromDistrict(districtUuids) {
+  const filters = [];
+  if (districtUuids) {
+    filters.push(`parent_Uuid_In: ["${districtUuids.join('", "')}"]`);
+  }
+  const payload = formatPageQuery("locations", filters, [
+    "id, uuid, code, name, parent { id, uuid, name, code }, children { edges {node {id, uuid, code, name, parent { id, uuid, code, name}}}}",
```

Comment:
id, uuid, code, name, parent {} seems to be repeatable. Can we simplify this by creating special const?

Reply:
I created special const DISTRICT_DATA_FETCH_PARAMS to don't repeat myself.